src/SignalR/server/Core/src/IHubContext.cs

```diff
@@ -18,11 +18,4 @@ public interface IHubContext
         /// </summary>
         IGroupManager Groups { get; }
     }
-
-    /// <summary>
-    /// A context abstraction for a hub.
-    /// </summary>
-    public interface IHubContext<out THub> : IHubContext where THub : Hub
```

Comment:
Keep the non generic interface and implement it on the concrete type so the original scenario is still possible.